### PR TITLE
Final updates

### LINKED ISSUE #154: Final updates
- [x] Make this logo clear on the nav bar. It's clear on the footer so check that out to see whats different there. I think it's the way we're passing in width + height but I'm not sure 
Screenshot 2023-02-03 at 4 27 54 PM

- [ ] Add in the add to calendar and the 3 hacker guide links (we really need to add these in this sprint since its the last sprint before the hackathon. So don't be afraid to track people down and emphasize that they really need to get these to us LOL) 
- [x] Make sure all the links on the navbar / footer link to the correct section (I think schedule doesn't link to anything right now) 
- [x] THe B in the HackBeanpot needs to be capitalized Screenshot 2023-02-03 at 4 32 55 PM
- [x] Add space between "at" and the url
Screenshot 2023-02-03 at 4 33 21 PM

### PR DESCRIPTION
**Pull request by Dhivas-Sugumar**
Final updates

Fixes #154 .

Changelist:

Added in all changes for the ticket. 

Tested:

- locally on dev environment.

### CODE REVIEW

File: live-site/lib/data.ts [hunk 1]
```diff
@@ -11,15 +11,15 @@ const liveSiteTabInfo: TabInfo[] = [
 const resources: ResourceBoxProps[] = [
   {
     label: 'Project Guide',
-    link: '#'
+    link: 'https://docs.google.com/document/d/1xhgMtE5XmauOgewQ4M9_rqQd4JQUZQ1TgBW2z-pZfa8/edit?usp=sharing'
```
Comment (jzhzhang66) [inline live-site/lib/data.ts]: I just talked with PL and we're switching this link to https://docs.google.com/document/d/1kob1aZ6VLQ0li-66onzF09WxdI9sjHPI-zHznm353tk/edit

to include the project submission guide! Sorry for the last minute change

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
Requested 1 change and otherwise just add in the calendar link from sarah whenever you get that. Thanks!!!

**Review by srazh — APPROVED** [review]
Approved and dmed u the link just now! sorry for the delay.

**Review by jzhzhang66 — APPROVED** [review]
LGTM

**Pull request merged**